<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>250</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="keyLabel">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>120</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Key File:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>119</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="unameLabel">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>45</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="certEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>94</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="unameEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>44</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passLabel">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="certLabel">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>95</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cert File:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>69</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="appKeyLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>145</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Application Key:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="appKeyEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>145</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="loadLoginFromFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Details From File</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>unameEdit</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>certEdit</tabstop>
  <tabstop>keyEdit</tabstop>
  <tabstop>appKeyEdit</tabstop>
  <tabstop>loadLoginFromFileButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
